The job `mingw-check` of your PR [failed](https://dev.azure.com/rust-lang/rust/_build/results?buildId=17992&view=logs&jobId=74054d28-c774-5fab-ecc2-352fe71b230e) ([pretty log](https://rust-lang.github.io/rust-log-analyzer/log-viewer/#https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/17992/logs/65), [raw log](https://dev.azure.com/rust-lang/e71b0ddf-dd27-435a-873c-e30f86eea377/_apis/build/builds/17992/logs/65)). Through arcane magic we have determined that the following fragments from the build log may contain information about the problem.

<details><summary><i>Click to expand the log.</i></summary>

```plain
2020-01-09T18:21:57.4773862Z ##[command]git remote add origin https://github.com/rust-lang/rust
2020-01-09T18:21:57.4863339Z ##[command]git config gc.auto 0
2020-01-09T18:21:57.4954161Z ##[command]git config --get-all http.https://github.com/rust-lang/rust.extraheader
2020-01-09T18:21:57.5028742Z ##[command]git config --get-all http.proxy
2020-01-09T18:21:57.5169928Z ##[command]git -c http.extraheader="AUTHORIZATION: basic ***" fetch --force --tags --prune --progress --no-recurse-submodules --depth=2 origin +refs/heads/*:refs/remotes/origin/* +refs/pull/68066/merge:refs/remotes/pull/68066/merge
---
2020-01-09T18:26:51.9814565Z     Checking panic_unwind v0.0.0 (/checkout/src/libpanic_unwind)
2020-01-09T18:26:56.9953112Z error: the feature `manually_drop_take` has been stable since 1.42.0 and no longer requires an attribute to enable
2020-01-09T18:26:56.9955545Z    --> src/libstd/lib.rs:278:12
2020-01-09T18:26:56.9956314Z     |
2020-01-09T18:26:56.9956947Z 278 | #![feature(manually_drop_take)]
2020-01-09T18:26:56.9958816Z     |
2020-01-09T18:26:56.9959516Z     = note: `-D stable-features` implied by `-D warnings`
2020-01-09T18:26:56.9959820Z 
2020-01-09T18:26:57.1484266Z error: aborting due to previous error
---
2020-01-09T18:26:57.1740185Z   local time: Thu Jan  9 18:26:57 UTC 2020
2020-01-09T18:26:57.4632250Z   network time: Thu, 09 Jan 2020 18:26:57 GMT
2020-01-09T18:26:57.4635094Z == end clock drift check ==
2020-01-09T18:26:58.7467296Z 
2020-01-09T18:26:58.7630996Z ##[error]Bash exited with code '1'.
2020-01-09T18:26:58.7657976Z ##[section]Starting: Checkout
2020-01-09T18:26:58.7659892Z ==============================================================================
2020-01-09T18:26:58.7659949Z Task         : Get sources
2020-01-09T18:26:58.7659999Z Description  : Get sources from a repository. Supports Git, TfsVC, and SVN repositories.
```

</details><p></p>

[I'm a bot](https://github.com/rust-ops/rust-log-analyzer)! I can only do what humans tell me to, so if this was not helpful or you have suggestions for improvements, please ping or otherwise contact **`@TimNN`**. ([Feature Requests](https://github.com/rust-ops/rust-log-analyzer/issues?q=is%3Aopen+is%3Aissue+label%3Afeature-request))

--- a/src/libcore/mem/manually_drop.rs
+++ b/src/libcore/mem/manually_drop.rs
@@ -87,27 +87,28 @@ impl<T> ManuallyDrop<T> {
         slot.value
     }
 
-    /// Takes the contained value out.
+    /// Takes the value from the `ManuallyDrop<T>` container out.
     ///
     /// This method is primarily intended for moving out values in drop.
     /// Instead of using [`ManuallyDrop::drop`] to manually drop the value,
     /// you can use this method to take the value and use it however desired.
-    /// `Drop` will be invoked on the returned value following normal end-of-scope rules.
     ///
-    /// If you have ownership of the container, you can use [`ManuallyDrop::into_inner`] instead.
+    /// Whenever possible, it is preferrable to use [`into_inner`][`ManuallyDrop::into_inner`]
+    /// instead, which prevents duplicating the content of the `ManuallyDrop<T>`.
     ///
     /// # Safety
     ///
-    /// This function semantically moves out the contained value without preventing further usage.
-    /// It is up to the user of this method to ensure that this container is not used again.
+    /// This function semantically moves out the contained value without preventing further usage,
+    /// leaving the state of this container unchanged.
+    /// It is your responsibility to ensure that this `ManuallyDrop` is not used again.
     ///
     /// [`ManuallyDrop::drop`]: #method.drop
     /// [`ManuallyDrop::into_inner`]: #method.into_inner
     #[must_use = "if you don't need the value, you can use `ManuallyDrop::drop` instead"]
-    #[unstable(feature = "manually_drop_take", issue = "55422")]
+    #[stable(feature = "manually_drop_take", since = "1.42.0")]
     #[inline]
     pub unsafe fn take(slot: &mut ManuallyDrop<T>) -> T {
-        ManuallyDrop::into_inner(ptr::read(slot))
+        ptr::read(&slot.value)
     }
 }
 

--- a/src/libstd/lib.rs
+++ b/src/libstd/lib.rs
@@ -275,7 +275,6 @@
 #![feature(link_args)]
 #![feature(linkage)]
 #![feature(log_syntax)]
-#![feature(manually_drop_take)]
 #![feature(maybe_uninit_ref)]
 #![feature(maybe_uninit_slice)]
 #![feature(needs_panic_runtime)]

